webpack/assets/javascripts/react_app/common/MountingService.js

```diff
@@ -5,6 +5,8 @@ import React from 'react';
 import store from '../redux';
 import componentRegistry from '../components/componentRegistry';
 
+export { default as store, registerReducer } from '../redux';
```

Comment:
I'm not sure if this is the best place to add these exports to expose them to plugins. I just took a shortcut since `foremanReact/common/MountingService` was already available.